src/test/java/com/smartbear/readyapi/client/PluginTestStepBuilderTest.java

```diff
@@ -0,0 +1,34 @@
+package com.smartbear.readyapi.client;
+
+import com.smartbear.readyapi.client.model.PluginTestStep;
+import org.junit.Test;
+
+import static com.smartbear.readyapi.client.TestRecipeBuilder.newTestRecipe;
+import static com.smartbear.readyapi.client.teststeps.TestSteps.pluginTestStep;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class PluginTestStepBuilderTest {
+    @Test
+    public void buildsRecipeWithPluginTestStep() throws Exception {
+        TestRecipe testRecipe = newTestRecipe()
+                .addStep(pluginTestStep("MQTTPublishTestStep")
```

Comment:
It's better this way. Passing strings is error-prone and hard to troubleshoot for users of the client, so we should only use this for plugin test steps.